manifests/filter/api_validator.pp

```diff
@@ -76,7 +80,8 @@
   $validators         = undef,
   $multi_role_match   = false,
   $version            = undef,
-  $delegating         = false
+  $delegating         = false,
+  $delegating_quality = undef
```

Comment:
I'd really prefer you always end these with a comma, so that when we add items, we dont have multiple lines changing.